auction/README.md

```diff
@@ -328,8 +319,11 @@ The output should look something like the following:
 
 ```
 --> Submit the transaction to end the auction
-2020-11-06T13:16:11.591Z - warn: [TransactionEventHandler]: strategyFail: commit failure for transaction "99feade5b7ec223839200867b57d18971c3e9f923efc95aaeec720727f927366": TransactionError: Commit of transaction 99feade5b7ec223839200867b57d18971c3e9f923efc95aaeec720727f927366 failed on peer peer0.org1.example.com:7051 with status ENDORSEMENT_POLICY_FAILURE
-******** FAILED to submit bid: TransactionError: Commit of transaction 99feade5b7ec223839200867b57d18971c3e9f923efc95aaeec720727f927366 failed on peer peer0.org1.example.com:7051 with status ENDORSEMENT_POLICY_FAILURE
+2021-01-28T16:47:27.501Z - error: [DiscoveryHandler]: compareProposalResponseResults[undefined] - read/writes result sets do not match index=1
+2021-01-28T16:47:27.503Z - error: [Transaction]: Error: No valid responses from any peers. Errors:
+    peer=undefined, status=grpc, message=Peer endorsements do not match
+******** FAILED to submit bid: Error: No valid responses from any peers. Errors:
+    peer=undefined, status=grpc, message=Peer endorsements do not match
```

Comment:
Does anyone know why instead of the explicit error we used to get we now get this rather cryptic one? @denyeart ?

Reply:
The error actually reflects what is happening in the sample. one organization is trying to close the auction, while another organization is rejecting the transaction due to a non disclosed bid. As a result, the endorsements from the two peers do not match.